tests/Fixtures.ts

```diff
@@ -39,27 +42,16 @@ export const parametrizedTLNetworkConfig = [
   { config: tlNetworkConfigIdentity, walletType: 'Identity' }
 ]
 
-export const user1 = {
-  address: '0xf8E191d2cd72Ff35CB8F012685A29B31996614EA',
-  keystore: keystore1,
-  pubKey: 'a5da0d9516c483883256949c3cac6ed73e4eb50ca85f7bdc2f360bbbf9e2d472'
-}
-
-export const user2 = {
-  address: '0xcE2D6f8bc55A61428D32947bC9Bc7F2DE1640B18',
-  keystore: keystore2
-}
-
-export const user3 = {
-  address: '0xc7D6401b81A2F70C5906e8A6FBEbE680c69fA03D',
-  keystore: keystore3,
-  pubKey: '3a1430b32ded788f4059ba31cbe6157ac8788f7198d4aed41e1091a97bff2857'
-}
-
 export const extraData = '0x12ab34ef'
 
-export function createUsers(tlInstances: TLNetwork[]) {
-  return Promise.all(tlInstances.map(tl => tl.user.create()))
+export async function createAndLoadUsers(tlInstances: TLNetwork[]) {
+  return Promise.all(
+    tlInstances.map(async tl => {
+      const createdUser = await tl.user.create()
+      await tl.user.load(createdUser.wallet)
```

Comment:
So you split up the creation and loading of an account on purpose? Why? I think it is intuitive that create account, restore account will not change the state, but load will.